en/flight_controller/snapdragon_flight_advanced.md

```diff
@@ -1,4 +1,5 @@
 # Snapdragon Advanced
+This page is a collection of legacy pieces of documentation which might come in handy when debugging on the Snapdragon platform.
```

Comment:
Lots of what looks like useful information here, but not filled with confidence by "might come in handy". Also this covers more than just debugging - there is camera setup information and other stuff too. 

Is this really just a mixed bag that you won't properly maintain? Or might you add a debugging section in future?

Reply:
Well, the lack of confidence stems from the fact that I haven't tried some of these things myself. I'll give the page a bit more credit by changing this line if you prefer.